.github/workflows/pr_test_cpu.yml

```diff
@@ -20,32 +20,32 @@ jobs:
         os: ['Ubuntu-latest', 'Windows-latest']
         pytorch-dtype: ['float32', 'float64']
 
-    uses: kornia/workflows/.github/workflows/tests.yml@v1.10.1
+    uses: kornia/workflows/.github/workflows/tests.yml@v1.11.0
     with:
       os: ${{ matrix.os }}
       python-version: '["3.8", "3.12"]'
-      pytorch-version: '["1.9.1", "2.3.1"]'
+      pytorch-version: '["1.9.1", "2.4.0"]'
       pytorch-dtype: ${{ matrix.pytorch-dtype }}
 
   tests-cpu-macos:
-    uses: kornia/workflows/.github/workflows/tests.yml@v1.10.1
+    uses: kornia/workflows/.github/workflows/tests.yml@v1.11.0
     with:
       os: 'MacOS-latest'
       python-version: '["3.8", "3.12"]'
       pytorch-dtype: 'float32'
 
 
   coverage:
-    uses: kornia/workflows/.github/workflows/coverage.yml@v1.10.1
+    uses: kornia/workflows/.github/workflows/coverage.yml@v1.11.0
 
   typing:
-    uses: kornia/workflows/.github/workflows/mypy.yml@v1.10.1
+    uses: kornia/workflows/.github/workflows/mypy.yml@v1.11.0
 
   tutorials:
-    uses: kornia/workflows/.github/workflows/tutorials.yml@v1.10.1
+    uses: kornia/workflows/.github/workflows/tutorials.yml@v1.11.0
```

Comment:
tutorials will fail because i forget to update the vision version -- in addition to update the version, will need to figure out a way to inject it because the "new torch repository" wheels -- the old one isn't working